format/FormatCBFMini.py

```diff
@@ -15,6 +15,7 @@
 from cctbx.eltbx import attenuation_coefficient
 from iotbx.detectors.pilatus_minicbf import PilatusImage
 from scitbx.array_family import flex
+from cctbx import factor_ev_angstrom
```

Comment:
```suggestion
from cctbx import factor_ev_angstrom
from cctbx.eltbx import attenuation_coefficient
from iotbx.detectors.pilatus_minicbf import PilatusImage
from scitbx.array_family import flex
```

please keep imports sorted